r_examples/r_api_serving_examples/API Serving Examples.ipynb

```diff
@@ -0,0 +1,881 @@
+{
```

Comment:
<p>Do you want to add a concluding section about which approach(es) of the three are recommended given runtimes or other aspects?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
I'll add a section addressing this.